tests/test_packet_bundling.c

```
@@ -0,0 +1,311 @@
+    ASSERT_EQ(out[4], 0x80);
+    /* priority body begins after [type][len][0x80][seqHi][seqLo] */
+    ASSERT_EQ(out[7], 0x06);
+}
+
+/* === Pass 3 promotion + dequeue === */
+
+TEST(drain_unreliable_promotes_and_frees)
+{
+    /* Pass 3 drains unreliable messages, dequeues+frees each, and reports the
+     * promoted ones to the caller. */
+    bc_drain_t d;
```

Clang (14.0.6)

[warning] variable name 'd' is too short, expected at least 3 characters

(readability-identifier-length)